driver/driver_default.go

```diff
@@ -13,6 +15,24 @@ type DefaultDriver struct {
 	r Registry
 }
 
+// IsSQLiteMemoryMode returns true if SQLite if configured to use memory mode
+func IsSQLiteMemoryMode(dsn string) bool {
+	/*
+		if urlParts := strings.SplitN(dsn, "?", 2); len(urlParts) == 2 && strings.HasPrefix(dsn, "sqlite") {
+			queryVals, err := url.ParseQuery(urlParts[1])
+			fmt.Println("2>>>>>> " + urlParts[1])
+			if err == nil && queryVals.Get("mode") == "memory" {
+				return true
+			}
+		}
+		return false
+	*/
```

Comment:
If you remove this this will be good to merge!

Reply:
sure, also spotted it and fixed it.